docker/cortx-build/Makefile

```diff
@@ -15,10 +15,17 @@ help : Makefile
 	@echo
 	@echo targets:
 	@sed -n 's/^##/   /p' $<
+
+validation:
+	@echo "*************** Validate build node *********************"
+    if [[ "$(df -BG  / | awk '{ print $4 }' | tail -n 1 | sed 's/G//')" -lt "65" ]]; then
```

Comment:
This command will be running inside the container. Please validate that the required volume will be available.

Reply:
Yes it's working:
![image](https://user-images.githubusercontent.com/91177647/179196982-4c24adf1-9420-4f95-92b7-ceca09c7ce12.png)

Reply:
Now checking for /cortx-workspaces since all build are happening there inside container. Screenshots attached above of tests.